mflowgen/global_controller/construct.py

```diff
@@ -119,6 +116,18 @@ def construct():
   init.extend_inputs( custom_init.all_outputs() )
   power.extend_inputs( custom_power.all_outputs() )
 
+  # Must adjust pin_cap_tolerance to prevent:
+  # **ERROR: (TECHLIB-1071): Identified pin capacitance mismatch
+  #   '0.015400' and '0.015300' in pin 'trst_n', for cell 'global_controller',
+  #   between libraries 'analysis_default.lib' and 'analysis_hold.lib'.
+
+  # Before: merge_model_timing ... -outfile design.lib
+  # Or:     merge_model_timing ... -outfile design.lib -pin_cap_tolerance .000001
+  # After:  merge_model_timing ... -outfile design.lib -pin_cap_tolerance .0005
+
+  cmd = "cat scripts/extract_model.tcl | sed 's/\(merge_model_timing.*design.lib\).*/\\1 -pin_cap_tolerance .0005/' > inputs/extract_model.tcl"
+  genlib.pre_extend_commands( [ cmd ] )
+
```

Comment:
It may be better to just try using pt for this block rather than inserting this command

Reply:
done. using pt instead of innovus now.